src/rust/catnap/futures/mod.rs

```diff
@@ -1,19 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-//==============================================================================
+//======================================================================================================================
```

Comment:
Not sure if this is intentional, but this file was only changed to extend the comment bars. I don't mind having style fixes in this PR, but it should probably have its own commit.

Reply:
I was following our agreement that if something would be off the "coding style" it could be cleaned up. Anyways, I have reverted these changes.

Reply:
Yes, that is fine but I just wanted a separate commit in the PR for the style changes.